shared/chat/manage-channels/container.tsx

```diff
@@ -74,19 +75,20 @@ const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
 }
 
 const mapDispatchToProps = (dispatch: Container.TypedDispatch, ownProps: OwnProps) => {
-  const teamname = Container.getRouteProps(ownProps, 'teamname', '')
+  const teamID = Container.getRouteProps(ownProps, 'teamID', Types.noTeamID)
```

Comment:
If I'm following properly, when `teamID` on the route props doesn't exist, we're defaulting to `noTeamID = NOTEAMID` and passing to `saveChannelMembership`.

Assuming this happens, the farthest along I've been able to trace is to `_joinConversation` where it will produce a black bar via `globalErorr`.

If we're going to use `noTeamID` when a `teamID` is missing, I wonder if bailing on the action and logging is a safer approach. Honestly not sure about this since the same situation was happening before, but the only difference is that `noTeamID` is **not falsey**.

Reply:
Actually after reviewing the other usages of `Container.getRouteProps(ownProps, 'teamID', ...)` it seems that defaulting to `Types.noTeamID` is the outlier, so I wonder what the intention is to use `noTeamID` over `''`.

Reply:
Seems like it black bars either way so ¯\_(ツ)_/¯